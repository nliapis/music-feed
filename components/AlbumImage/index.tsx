import { IMImage } from '../../interfaces';
import styles from './index.module.scss';

export default function AlbumImage({ label, attributes }: IMImage) {
  return <img src={label} {...attributes} alt="" className={styles.image} />;
}
