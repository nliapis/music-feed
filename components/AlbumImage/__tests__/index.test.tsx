import React from 'react';
import renderer from 'react-test-renderer';
import AlbumImage from '../index';

it('renders AlbumImage', () => {
  const props = {
    label:
      'https://is3-ssl.mzstatic.com/image/thumb/Music113/v4/59/7e/51/597e5166-d6c8-ac39-8217-74d6cd9baaa6/19UMGIM94144.rgb.jpg/170x170bb.png',
    attributes: {
      height: '170',
    },
  };
  const tree = renderer.create(<AlbumImage {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
