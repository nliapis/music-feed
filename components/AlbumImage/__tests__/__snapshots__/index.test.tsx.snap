// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AlbumImage 1`] = `
<img
  alt=""
  className="image"
  height="170"
  src="https://is3-ssl.mzstatic.com/image/thumb/Music113/v4/59/7e/51/597e5166-d6c8-ac39-8217-74d6cd9baaa6/19UMGIM94144.rgb.jpg/170x170bb.png"
/>
`;
